### PR TITLE
Fix typos and update doc.

### PR DESCRIPTION
**Pull request by csukuangfj**

### CODE REVIEW

File: docs/parallelism.rst [hunk 1]
```diff
@@ -31,12 +31,12 @@ An example of use to extract features on a :class:`lhotse.CutSet`:
 .. code-block::
 
     from concurrent.futures import ProcessPoolExecutor
-    from lhotse import CutSet, Fbank, LilcomFilesWriter
+    from lhotse import CutSet, Fbank, ChunkedLilcomHdf5Writer
```
Comment (pzelasko) [inline docs/parallelism.rst:34]: is there a reason you prefer the chunked HDF5 version to the default `LilcomChunkyWriter`?

**Review by pzelasko — COMMENTED** [review]
That’s a lot of typos, thanks!

File: docs/parallelism.rst (hunk 1 above)
Comment (csukuangfj) [reply docs/parallelism.rst]: No, I did not realize that there is a `LilcomChunkyWriter`.
Just reverted to `LilcomChunkyWriter`.

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**